### PR TITLE
Add background-blend-mode and mix-blend-mode

### PR DESCRIPTION
**Pull request by jamesnw**
The `lighter-plus` and `darker-plus` did not seem to garner enough community interest to warrant separate features or block baseline.

Also, `css.types.blend-mode` are shared between mix- and background-, so I put them with the earlier feature, but `lighter-plus` and `darker-plus` are only available on mix-.

For both mix- and background-, the Safari versions are slightly different between [Caniuse](https://caniuse.com/css-backgroundblendmode) and BCD- is this a candidate for https://github.com/web-platform-dx/web-features/issues/1499?

Part of #1624

### CODE REVIEW

File: groups/blend-mode.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# https://drafts.fxtf.org/compositing-1/
+# Blend modes control how parts of an element are composited, or visually mixed.
+name: Blend mode
```
Comment (captainbrosset) [inline groups/blend-mode.yml:3]: I think this should be a child of CSS (i.e. add `parent: css`).

File: features/background-blend-mode.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+name: background-blend-mode
+description: The `background-blend-mode` CSS property sets how an element's background is visually mixed.
```
Comment (captainbrosset) [inline features/background-blend-mode.yml:2]:
```suggestion
description: The `background-blend-mode` CSS property blends an element's background image and background color.
```

I've never used this so I don't know whether this is correct, but MDN seems to say it is.

File: features/mix-blend-mode.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+name: mix-blend-mode
+description: The `mix-blend-mode` CSS property sets how an element's content is visually mixed.
```
Comment (captainbrosset) [inline features/mix-blend-mode.yml:2]: How about:

```suggestion
description: The `mix-blend-mode` CSS property blends an element's content with its background or parent elements.
```

**Review by captainbrosset — CHANGES_REQUESTED** [review]

File: features/mix-blend-mode.yml (hunk 3 above)
Comment (jamesnw) [reply features/mix-blend-mode.yml:2]: I like the "blends" verb, so I accepted the change. I also added example blend modes to convey that there are options.

**Review by ddbeck — APPROVED** [review]
I think this is ready to go now. Thank you!

**Pull request merged**